src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -417,12 +416,12 @@ static int32_t EnumAbbrevEraNames(const char* locale,
     // The C-API for ICU provides no way to get at the abbreviated era names for a calendar (so we can't use EnumSymbols
     // here). Instead we will try to walk the ICU resource tables directly and fall back to regular era names if can't
     // find good data.
-    char localeNameBuf[ULOC_FULLNAME_CAPACITY];
-    char parentNameBuf[ULOC_FULLNAME_CAPACITY];
+    char localeNameBuf[ULOC_FULLNAME_CAPACITY]{};
+    char parentNameBuf[ULOC_FULLNAME_CAPACITY]{};
 
     char* localeNamePtr = localeNameBuf;
     char* parentNamePtr = parentNameBuf;
-    STRING_COPY(localeNamePtr, sizeof(localeNameBuf), locale);
+    STRING_COPY(localeNamePtr, sizeof(localeNameBuf), locale)
```

Comment:
You are right that `;` is technically not required here given how to macro is implemented today.  I think it is better with `;`. It makes it easier to see that there are no unintended interactions.

Reply:
Ok. I will get that `;` added here.